frontend/tsconfig.json

```diff
@@ -18,7 +18,8 @@
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react-jsx",
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    "baseUrl": "src"
```

Comment:
This seems out of scope for this PR as it would affect how we import things app-wide. I have no problem switching to absolute imports but I think it'd be better as a separate PR so folks can discuss.

Reply:
Got it. Will remove this, from this PR.